manager.go

```diff
@@ -8,57 +8,58 @@ import (
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+// ValidationResult describes the result of validating a voucher
+type ValidationResult struct {
+	// Accepted indicates where the request was accepted if a request is not
+	// accepted, the request fails. This is true for revalidation as well
+	Accepted bool
+	// VoucherResult provides information to the other party about what happened
+	// with the voucher
+	VoucherResult
+	// LeaveRequestPaused indicates whether the request should stay paused
+	// even if the request was accepted
+	LeaveRequestPaused bool
```

Comment:
I suggest renaming this to Paused

Reply:
ultimately changed to ForcePause in subsequent PR